Code4SocialGood/c4sg-web
# Organization Detail Page - Display Logo

**Notes**
- Resubmission of #140 with changes to `organization.service` and `OrganizationEditComponent` to use the new endpoints
> - Resolves #47 and prepares for future image changes
> - An `UploaderService` is created to conveniently increase manageability, reduce boilerplate for future image changes, and further loosely couple `OrganizationService.getLogo` and `Organization.uploadLogo` from browser file upload operations in case of future changes in back end

**To test:** 
1. Make sure back end is synced to latest commit
2. Login as nonprofit user: testnonprofituser2@gmail.com / Opensource5social!
3. Navigate to "My Organization" or `http://localhost:4200/nonprofit/edit/1`
4. Upload an image
5. Navigate to `http://localhost:4200/nonprofit/view/1`
